SCClassLibrary/Common/GUI/Base/QDialog.sc

```diff
@@ -40,13 +43,13 @@ FileDialog : QObject {
 
 Dialog {
 
-	*openPanel { arg okFunc, cancelFunc, multipleSelection=false;
+	*openPanel { arg okFunc, cancelFunc, multipleSelection=false, directory;
 		var fileMode;
 		if( multipleSelection ) { fileMode = 3 } { fileMode = 1 };
-		^FileDialog.new( okFunc, cancelFunc, fileMode, 0, stripResult:multipleSelection.not );
+		^FileDialog.new( okFunc, cancelFunc, fileMode, 0, stripResult:multipleSelection.not, directory: directory );
```

Comment:
the args are already in correct order, so you could remove `stripResult:` and `directory:` here for simplicity

Reply:
… also for speed.